ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -50,6 +50,7 @@ Install RHODS
   END
 
 Verify RHODS Installation
+  [Arguments]  ${cluster_type}
```

Comment:
not needed as mentioned above

Reply:
Well, it's used also in a keyword above, so let's keep it to make things consistent. Also, it's always better prefer the passing arguments over having too many global, but I understand you.

Reply:
I removed the argument before reading this comment. I'll keep it without the argument